examples/sentencepiece/train.py

```diff
@@ -0,0 +1,156 @@
+import torch
```

Comment:
Instead of copy-pasting train.py here you could modify the other examples to use the sentencepiece abstraction you implemented in the main library. That probably makes more sense to do after you migrate current tokenizers to the transform / functional split.